presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/DetermineJoinDistributionType.java

```diff
@@ -91,22 +98,87 @@ public static boolean canReplicate(JoinNode joinNode, Context context)
         return buildSideSizeInBytes <= joinMaxBroadcastTableSize.toBytes();
     }
 
+    public static double getSourceTablesSizeInBytes(PlanNode node, Context context)
+    {
+        return getSourceTablesSizeInBytes(node, context.getLookup(), context.getStatsProvider(), context.getSymbolAllocator().getTypes());
+    }
+
+    @VisibleForTesting
+    static double getSourceTablesSizeInBytes(PlanNode node, Lookup lookup, StatsProvider statsProvider, TypeProvider typeProvider)
```

Comment:
these are utilities, move them below where they are used

Reply:
The is one uitility above this one (`canReplicate`) and more utilities below, so i am not quite convinced. Up to you